### ISSUE TITLE
[0.2.1-Linux] Lobibox causes endless loading spinny

### ISSUE THREAD

**Issue by RealGrep** [jeperi/evelscan#1]
I'm actually on the latest Git version, which incorporates the "Auto works now" changes and the readme changes (but nothing after, if you add more changes). On Arch Linux.

What happens is I start it, it fades to a "Loading..." with the spinny, and stays there forever. Maybe an issue with Lobibox on Linux? No idea. However, I traced it down to that code way at the bottom of the eve.js file.

If I comment out the entire block that starts with `Lobibox.alert('error', {` and so on near the end of the file, all the way to after `$("#spinnerDimmer").show();` and add the lines from inside it after the newly commented block:
```
a.blanked = false;
versionCheck();
```

Should be around line 518.

That makes it work just fine. Just that one block seems to, for whatever odd reason, not work for me.

By the way, looking very good. Much more compact. Still needs the colour coding, and I'm sure other things, but I like where you're going with it. (And just a quick plug for being able to exclude entries from players in my own corp and/or alliance, because that'd be quite handy.)

Oh yeah, and it was me who logged that issue with xsel being required in Linux, in case you were wondering. Keep up the fantastic work! <3

**Comment by jeperi** [comment]
Thanks,

that makes me wonder if the other dialogs won't show up too. I have Arch on my laptop, I'll do some testing on it. That alert box is probably not very compulsory, so I might get rid of it anyway.

It's my first time coding something apart from simple show/hide type work with javascript, which seems to have its share of quirks, and the code is still a bit of a mess. Still, it's getting better.

I'll try to set up the color coding. It's a bit harder now that I'm using slickgrid, but I found a method that can apply a custom css class to a specific column on a specific row. I could use that and dump a set of custom classes to html on demand. Or then just do it manually.

I looked into how the SSO thingie CCP provides works. Apparently the xml and crest APIs are going to be deprecated in a year or two, so that's probably the way to go. Excluding entries from players in your own corp shouldn't be very hard. I'll probably implement that soon.
